gossip/src/crds.rs

```diff
@@ -678,20 +678,22 @@ impl CrdsDataStats {
             }
         }
 
-        if should_report_message_signature(&entry.value.signature) {
-            datapoint_info!(
-                "gossip_crds_sample",
-                (
-                    "origin",
-                    entry.value.pubkey().to_string().get(..8),
-                    Option<String>
-                ),
-                (
-                    "signature",
-                    entry.value.signature.to_string().get(..8),
-                    Option<String>
-                )
-            );
+        if let GossipRoute::PushMessage = route {
+            if should_report_message_signature(&entry.value.signature) {
```

Comment:
why `if let` here?
This could be
```rust
if matches!(route, GossipRout::PushMessage) 
    && should_report_message_signature(&entry.value.signature) {
```

Reply:
good point. will update